tests/widget/inline/widget-inline.webauthn-login.test.js

```diff
@@ -0,0 +1,44 @@
+import { expect, test } from '@playwright/test';
+import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
+
+test('inline widget with webauthn login', async ({ browser, page }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
+    const cdpSession = await page.context().newCDPSession(page);
+
+    await cdpSession.send('WebAuthn.enable');
+    authenticator = await cdpSession.send('WebAuthn.addVirtualAuthenticator', {
+      options: {
+        protocol: 'ctap2',
+        transport: 'internal',
+        hasUserVerification: true,
+        isUserVerified: true,
```

Comment:
Question - if this is false, does it "fake the finger print, but then fail the login"?